nostalgia/sources/web/videos_watched.py

```diff
@@ -24,4 +24,4 @@ def load(cls, nrows=None):
         )
         data["image"] = [get_thumbnail(x) for x in data["loc"]]
 
-        return cls(data.reset_index())
+        return cls(data.reset_index(drop=True))
```

Comment:
What does this line do?  (Forgive my inexperience with dataframes.)

Reply:
It means: "ignore the index" that pandas internally (and we) use. In our case we index time data by a TimeInterval index so we can do quick time lookups.

Reply:
Is it silly to ask why we're ignoring the index (or resetting, for that matter)?

Reply:
Because it has a time interval index and that cannot be stored to disk. On loading it is reconstructed.